components/brave_new_tab_ui/components/default/stats/style.ts

```diff
@@ -56,10 +52,10 @@ export const StyledStatsItemText = styled('span')<{}>`
   letter-spacing: 0;
 `
 
-export const StyledStatsItemDescription = styled('div')<{}>`
+export const StyledStatsItemDescription = styled('div')<{overriddenTextColor?: string}>`
   font-size: 16px;
   font-weight: 500;
-  color: #FFFFFF;
+  ${p => !p.overriddenTextColor && css`color: #FFFFFF;`}
```

Comment:
Optional: I think falsy values are removed by styled-components, so you could also write this as

```suggestion
  color: ${p => !p.overriddenTextColor && '#FFFFFF;'}
```